src/ui/zif_abapgit_popups.intf.abap

```diff
@@ -99,10 +99,10 @@ INTERFACE zif_abapgit_popups
       zcx_abapgit_exception .
   METHODS popup_transport_request
     IMPORTING
-      !is_transport_type  TYPE zif_abapgit_definitions=>ty_transport_type
-      !iv_use_default     TYPE abap_bool DEFAULT abap_false
+      !is_transport_type        TYPE zif_abapgit_definitions=>ty_transport_type
+      !iv_use_default_transport TYPE abap_bool DEFAULT abap_true
```

Comment:
previously: default = abap_false, `AND iv_use_default = abap_true` ie. it will never enter the IF

now: default = abap_true, `iv_use_default_transport = abap_true`, ie. by default it will enter the IF?

Reply:
now I got it 🤣